Add new known issue to 8.6 agent release notes (#2525)
elastic/observability-docs: docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc

Be aware that the live query results shown in {kib} may be delayed by up to 5 minutes.
====

[[known-issue-issue-2086]]

> .Beats started by agent can be configured without a valid output

Reviewer: ```suggestion
.Beats started by agent may fail with `output unit has no config` error
```
I think the original could be misread because the passive voice makes it unclear as to what is doing the configuring (user or system). I suggest including the error message because that's what users might search on.

Author: Thanks, this is much clearer

[%collapsible]
====
*Details* +

> A known issue in {agent} may lead to Beat processes being started without a
> valid output. The problem can be corrected by triggering a restart of {agent}
> or the affected Beats. Beats managed by {agent} can be restarted by changing the
> {agent} log level or the output section of the {agent} policy.
> For more information, refer to {agent-issue}2086[#2086].

Reviewer: ```suggestion
A known issue in {agent} may lead to Beat processes being started without a
valid output. To correct the problem, trigger a restart of {agent}
or the affected Beats. For Beats managed by {agent}, you can trigger a restart by changing the
{agent} log level or the output section of the {agent} policy.
For more information, refer to {agent-issue}2086[#2086].
```
Just trying to remove a bit of the passive voice.
